tasmota/tasmota_xdrv_driver/xdrv_01_9_webserver.ino

```diff
@@ -3356,7 +3356,8 @@ int WebQuery(char *buffer) {
         if (http_code == HTTP_CODE_OK || http_code == HTTP_CODE_MOVED_PERMANENTLY) {
 #ifdef USE_WEBSEND_RESPONSE
           // Return received data to the user - Adds 900+ bytes to the code
-          const char* read = http.getString().c_str();  // File found at server - may need lot of ram or trigger out of memory!
+          String response = String(http.getString().c_str());  // File found at server - may need lot of ram or trigger out of memory!
```

Comment:
Does it work too with just ``String response = http.getString()``?

Reply:
looks like yes. 15 minutes without error